nl/mail/chrome/messenger/preferences/advanced.dtd

```diff
@@ -128,8 +128,8 @@
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY showNextMessage.label          "Next Message">
-<!ENTITY showPreviousMessage.label      "Previous Message">
-<!ENTITY deleteSelectOrder.label        "After deleting a message select the:">
+<!ENTITY showNextMessage.label          "Volgend Bericht">
+<!ENTITY showPreviousMessage.label      "Vorig Bericht">
+<!ENTITY deleteSelectOrder.label        "Select na het verwijderen van het bericht de:">
```

Comment:
CORRECTION NEEDED, it should be: "Selecteer na het verwijderen van een bericht de:"